docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.1'
+services:
+  ogame:
+    container_name: ogame
+    build: .
+    environment:
+      - OGAMED_UNIVERSE=${OGAMED_UNIVERSE}
+      - OGAMED_USERNAME=${OGAMED_USERNAME}
+      - OGAMED_PASSWORD=${OGAMED_PASSWORD}
+      - OGAMED_LANGUAGE=${OGAMED_LANGUAGE}
+      - OGAMED_HOST=${OGAMED_HOST}
+      - OGAMED_PORT=${OGAMED_PORT}
+      - OGAMED_AUTO_LOGIN=${OGAMED_AUTO_LOGIN}
+      - OGAMED_PROXY=${OGAMED_PROXY}
+      - OGAMED_PROXY_USERNAME=${OGAMED_PROXY_USERNAME}
+      - OGAMED_PROXY_PASSWORD=${OGAMED_PROXY_PASSWORD}
+      - OGAMED_PROXY_TYPE=${OGAMED_PROXY_TYPE}
+      - OGAMED_PROXY_LOGIN_ONLY=${OGAMED_PROXY_LOGIN_ONLY}
+      - OGAMED_NEW_HOSTNAME=${OGAMED_NEW_HOSTNAME}
+      - OGAMED_AUTH_USERNAME=${OGAMED_AUTH_USERNAME}
+      - OGAMED_AUTH_PASSWORD=${OGAMED_AUTH_PASSWORD}
+      - OGAMED_ENABLE_TLS=${OGAMED_ENABLE_TLS}
+      - OGAMED_TLS_CERTFILE=${OGAMED_TLS_CERTFILE}
+      - OGAMED_TLS_KEYFILE=${OGAMED_TLS_KEYFILE}
+      - OGAMED_COOKIES_FILENAME=${OGAMED_COOKIES_FILENAME}
+      - CORS_ENABLED=${CORS_ENABLED}
+    ports:
+      - "8080:8080"
```

Comment:
Not sure about it and it's meaning with IP address, maybe this is the default behaviour because I never saw this kind of usage in production environment.

I don't know if it needs for those host machines without a loopback interface or maybe for those with more than one interface with the goal to bind the ports to a specific one.

I believe we don't need it!

Reply:
https://stackoverflow.com/a/46220742/4196220

Reply:
Yes, like I said when you have more than one network interfaces.

But I still believe this is not a problem since:

> This is less than desirable if you have a public IP address, or your machine has an IP on a large network.

Most of us should setup specific port forwarding rule even if the port is mapped on the interfaces in order to expose it.

For instance I have a lot of services (as docker container) on my raspberry and they are not accessible from the "world" if I don't add a port forwarding rule or in my case if I don't add a rule on my nginx reverse proxy.